src/cljs/snapshot/lumo/repl.cljs

```diff
@@ -1370,6 +1364,7 @@
                                     (merge opts {:ns (symbol ns-sym)})
                                     (fn [{:keys [ns value error] :as ret}]
                                       (try
-                                        (apply value socket fn-args)
+                                        ;; TODO: do we wanna splice args?
+                                        (value socket fn-args)
```

Comment:
Now that I'm thinking about it, we should probably switch `value` to `fn-symbol` or something that's representative of what it holds.